web-services/query/src/main/java/datawave/webservice/query/runner/QueryExecutorBean.java

```diff
@@ -437,7 +466,7 @@ private QueryData validateQuery(String queryLogicName, MultivaluedMap<String,Str
 
         log.debug(queryParameters);
         qp.clear();
-        qp.setRequestHeaders(httpHeaders != null ? httpHeaders.getRequestHeaders() : null);
+        qp.setRequestHeaders(httpHeaders != null ? MapUtils.toMultiValueMap(httpHeaders.getRequestHeaders()) : null);
```

Comment:
I'm not happy with this MapUtils stuff.  I will take another look at that.

Reply:
ehhhh, after looking at the usages of this class (MapUtils) I do think it's still necessary...